openapi.yml

```
@@ -1786,6 +1812,33 @@ paths:
       operationId: getInvites
       summary: List all Invites
       description: Gets a list of all `Invite` entities.
+  /invites/batch_delete:
+    post:
+      summary: Delete Multiple Invites
+      tags:
+        - Invites
+        - Batches
+      responses:
+        '202':
+          description: Accepted
+          content:
+            application/json:
+              schema:
+                $ref: ./models/BatchJob.v1.yaml
+        '401':
```

This should be a wildcard, too:

```suggestion
        '4XX':
```